view/frontend/templates/breadcheckout/price.phtml

```diff
@@ -0,0 +1,150 @@
+<?= /* @noEscape */ $block->getBeforeHtml(); ?>
+
+<div class="splitpay-clickable-price" style="padding-bottom: 5px"></div>
+
+<script type="text/javascript">
+    require(['jquery', 'Bread_BreadCheckout/js/product/splitpay'], function($, splitpay) {
+
+        document.splitpayconfigureButton = function (items) {
+            var buttonCss           = '<?= /* @noEscape */ $block->getButtonDesign(); ?>';
+            var financingInfo   = <?= /* @noEscape */ $block->getFinancingJson(); ?>;
+            var breadConfig = {
+                <?php if (!$block->isHealthcare()) : ?>
+                items: items,
+                <?php else : ?>
+                customTotal: items[0]['price'],
+                <?php endif; ?>
+                discounts: <?= /* @noEscape */ $block->getDiscountDataJson(); ?>,
+                allowCheckout: false,
+                done: function() {},
+
+                /**
+                 * Add to cart call back used when allow checkout from product page = false
+                 *
+                 * @param callback
+                 */
+                addToCart:  function(callback) {
+                    document.getElementById('product_addtocart_form').submit();
+                }
+            };
+
+            if (buttonCss !== null && buttonCss !== '' && buttonCss != false) {
+                breadConfig.customCSS = buttonCss;
+            }
+
+            if(financingInfo.enabled && financingInfo.mode.cart){
+                var cartSizeFinancingId = financingInfo.id;
+                var cartSizeThreshold = financingInfo.threshold;
+                var itemsPriceSum = items.reduce(function(sum, item) { return sum + item.price * item.quantity; }, 0) / 100;
+                breadConfig.financingProgramId = (itemsPriceSum >= cartSizeThreshold) ? cartSizeFinancingId : 'null';
+            } else if (financingInfo.enabled && financingInfo.mode.sku){
+                var productSku = "<?= /* @noEscape */ $block->getProduct()->getSku(); ?>";
+                if(financingInfo.sku_limit.includes(productSku) === true){
+                    breadConfig.financingProgramId = financingInfo.id;
+                }
+            }
+
+            if (!document.splitpayhasCheckedConfigData) {
+
+                var configDataUrl = '<?= /* @noEscape */ $block->getConfigDataUrl(); ?>';
+                $.ajax({
+                    url: configDataUrl,
+                    type: 'post',
+                    context: this
+                }).done(function(data) {
+                    document.splitpayhasCheckedConfigData = true;
+
+                    if (data.shippingContact !== false) {
+                        breadConfig.shippingContact = data.shippingContact;
+                    }
+
+                    if (data.billingContact !== false) {
+                        breadConfig.billingContact = data.billingContact;
+                    }
+
+                    if (typeof bread !== 'undefined') {
```

Comment:
is there a reason for checking `typeof bread` specifically rather than just `bread` ?